README.md

```diff
@@ -88,10 +89,12 @@ points if there's a gist or existing repo we can look at with you.
    What components do you want to see in the Windows App SDK? For example you can propose a new API on an existing type, or an idea for a new API.
    It's fine if you don't have all the details: you can start with a summary and rationale.
 
-**Over time, we will add Windows App SDK code and samples into this repo and provide code contribution/developer guidelines.**
+**Over time, we will add Windows App SDK code into this repo and provide code contribution/developer guidelines.**
 
 Read more about the [contribution guide here](docs/contributor-guide.md).
 
+Discover samples in the [WindowsAppSDK-Samples](https://github.com/microsoft/WindowsAppSDK-Samples) repository to assist in learning how to use the latest features!
```

Comment:
How about we (1) rename the `Documentation` header to be `Helpful links` (it isn't all just documentation already), and then (2) make Samples an entry under that list alongside the documentation bullet you added :)

Having samples along the contributing felt a bit off, probably makes more sense in helpful/quick links :)